data/hai/hai missions.txt

```diff
@@ -200,14 +200,33 @@ mission "Returning Home"
 			`You are stopped by a human in the <origin> marketplace. He takes off his hat and asks, "You're from the outside, aren't you?"`
 			choice
 				`	"Yes, I am."`
+					to display
+						not "hai start"
 					goto outsider
+				`	"Not really, I grew up here."`
+					to display
+						has "hai start"
+					goto clothe
 				`	"What do you mean 'the outside?'"`
 			
 			`	"Outside of Hai space. You don't look quite like most humans who live here, unless there's some new fashion trend going on that I don't know about."`
 			choice
 				`	"Oh, yes. I do come from outside."`
+					to display
+						not "hai start"
+				`	"Not really, I grew up here and made these clothes for myself."`
+					to display
+						has "hai start"
+					goto outsider
 			
+			label clothe
+			`	"That's weird, your clothes don't look like most do here." He says.`
+				to display
+					has "hai start"
 			label outsider
+			`	You explain to him that you grew up making clothes, meaning you eventually grew tired of the usual clothing you kept making. Eventually you tried to make some more original ones, including the one you're wearing right now. You add that whilst you may not have been born in human space, your family made sure you learned about the origins of your species; you know human space like your pocket.`
```

Comment:
I don't like the repeated `making` and `eventually`.

Reply:
```suggestion
			`	You explain to him that you grew up making clothes, meaning you eventually grew tired of the usual clothing you kept making. Once you gained enough experience you started creating more original ones, including the one you're wearing right now. You add that whilst you may not have been born in human space, your family made sure you learned about the origins of your species; you know human space like your pocket.`
```

Reply:
Much better.